katcp/test/test_resource_client.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
OK, I did add it to all files, will probably help slightly to prevent introducing subtle Py2-isms until we are fully ported.